css/main.css

```diff
@@ -181,6 +181,51 @@ nav .user-greet .user-greet-msg {
   margin: 0 10px;
 }
 
+.description-what-is {
+  background: #041187;
+  margin: 0 -25px;
```

Comment:
Why negative margin though? I suppose this is done to increase the width of the container. Why not use `width` property instead?

Reply:
content-wrapper has a margin of 25px that restricts to fill the bg to complete width, that's why I have used -ve margin to fill it completely. If there is another way to achieve this, please suggest.

Reply:
The margin provided by the parent container is used by other elements on the page too
Removing that margin from the parent container just for this container would mean that we'll have to add margins to all other elements manually, which means repetition of code and a lot of unnecessary efforts
Since this particular element doesn't need margin, we've used negative margin